File: public/phase-2-20160620022203/C/containing-number.c
// Name : Containing Number
#include <stdio.h>
int main() {
	// input:
	// n : count of the following numbers
	// k : k
	// numbers_to_check : 
	int n, k, numbers_to_check[100][1], result=2, idx, jdx;
	char numbers_to_check_tmp[1];
	scanf("%d %d", &n, &k);
	for(idx = 0; idx< n; idx++) {
		scanf("%s", &numbers_to_check_tmp[0]);
		for(jdx = 0;jdx<1;jdx++) {
			numbers_to_check[idx][jdx] = numbers_to_check_tmp[jdx];
		}
	}


	// write your code here
	// store your results in `result`

	// output
	printf("%d", result);
	return 0;
}

File: public/phase-2-20160620022203/PYTHON/containing-number.py
#!/usr/bin/python
# Name : Containing Number
# input:
# n : count of the following numbers
# k : k
# numbers_to_check : 
elements = raw_input().strip().split(' ')
n = int(elements[0])
k = int(elements[1])
numbers_to_check = []
for _ in xrange(n):
	numbers_to_check_elements = raw_input().strip().split(' ')
	numbers_to_check.append(map(int, numbers_to_check_elements))


# write your code here
# store your results in `result`

# output

# Dummy Data
result = 2

print(result)
